cmd/kubectl-delivery/deliver_kubectl.sh

```diff
@@ -22,3 +22,42 @@ fi
 : ${TARGET_DIR:?"Need to set TARGET_DIR, e.g. /opt/kube"}
 
 cp $(which kubectl) ${TARGET_DIR}
+
```

Comment:
What's the purpose of the script here?

Reply:
to check the worker  is running, if the worker is in other state, the init contianer will be in loop,just waiting for the worker running